pom.xml

```diff
@@ -0,0 +1,703 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
lewl the maven URLs lead to 404s

Reply:
Blame Maven @mistermoe hahahaha: https://maven.apache.org/pom.html